bin/fypp

```diff
@@ -2496,18 +2561,22 @@ class Fypp:
     def __init__(self, options=None, evaluator_factory=Evaluator,
                  parser_factory=Parser, builder_factory=Builder,
                  renderer_factory=Renderer):
-        syspath = self._get_syspath_without_scriptdir()
-        self._adjust_syspath(syspath)
         if options is None:
             options = FyppOptions()
+        syspath = self._get_syspath_without_scriptdir()
+        lookuppath = []
+        if options.moduledirs is not None:
+            lookuppath += [os.path.abspath(moddir) for moddir in options.moduledirs]
+        lookuppath.append(os.path.abspath('.'))
+        lookuppath += syspath
+        self._adjust_syspath(lookuppath)
```

Comment:
This is somewhat problematic, because it would change the sys.path permanently for the rest of the code. Originally, the syspath manipulation was in effect during the import of the user specified modules only. Is there any reason, why you want to make the changes permanent instead?